modules/pubsub/outputs.tf

```diff
@@ -47,3 +47,19 @@ output "topic" {
     google_pubsub_topic_iam_binding.default
   ]
 }
+
+output "schema_id" {
+  description = "Schema resource id."
+  value       = google_pubsub_schema.default[0].id
+  depends_on = [
+    google_pubsub_schema.default
+  ]
```

Comment:
- you need to wrap the value in a `try()` otherwise this will fail when `var.schema` is null. Same for the other output
- the `depends_on` is not needed
- can you sort the outputs and variables alphabetically?

Reply:
Done, thanks for the tips! I was actually wondering how to make the output "optional".